tasks/main.yml

```diff
@@ -133,6 +133,17 @@
 - name: print drush output
   debug: var=drush_output.stdout_lines
 
+- name: run entity updates.
```

Comment:
@nathandentzau Is this the right place in the order for this task? I would have assumed it belonged closer to the database update task, before config import.

Reply:
I wasn't really sure where to put it because this is not the order of operations I would choose myself when deploying a Drupal site. I think the following should occur in the order their listed:

1. **Cache clear**: Ensure the correct configuration split settings are applied.
2. **Configuration import**: Import configuration from file system
3. **Database updates**: Run database, config and schema updates
4. **Entity updates**: Run entity schema updates (this will also trigger a cache clear)

The reasoning behind this is many times contrib modules will alter existing configuration entities and entity schemas through `hook_update_N()`. We use Violinist on projects we also use this role on. I would think there are edge cases where when we merge in a module update made by Violinist that is altering config in their update hooks that those changes are being reverted when we do the two config imports in this task.

Reply:
You've opened a can of worms now. 😁 You think config import should come before database update? What about if there are schema changes a that the new config is tied to? It seems like `updb` should happen early on, no?

If we are going to end up reordering steps I think we should do it incrementally ie. not in this single PR, so I am happy to approve this code as is.

Reply:
Yeah I don't see a scenario where importing configuration first could adversely affect the database updates step, but there is for sure a scenario of the opposite as I've pointed out. I agree any changes to the deployment order should be rolled out incrementally. So I'm good to merge this in if you are.